app/assets/javascripts/application.js

```diff
@@ -558,7 +574,9 @@ function disableButtonToggle(item, explicit) {
   $(item).blur();
 }
 
-function icon(name) {
+function icon_text(name, inner_text, icon_class) {
   "use strict";
-  return '<span class="glyphicon glyphicon-' + name + '" />';
+  var icon = '<i class="' + icon_class + " " + icon_class + "-" + name + '"/>'
```

Comment:
@tbrisker i was not sure what is the outcome of the discussion, should we be using ```<i>``` or ```<span>``` ?